apps/nowcasting-app/components/helpers/mapUtils.ts

```diff
@@ -52,3 +52,11 @@ export const setActiveUnitOnMap = (mapContainer: HTMLDivElement | null, unit: Ac
 
 export const getActiveUnitFromMap = (map: mapboxgl.Map) =>
   (map.getContainer().dataset.unit as ActiveUnit) || "MW";
+
+export const getBoundingBoxFromPoint = (point: mapboxgl.Point, hitTolerance = 5) => {
```

Comment:
I wonder if the tolerance should be a bit more. (I think this is related)

When i hover over the text labels, I have to be in the top right corner for anything to show up, but if my mouse is on the centre, then no level appears. This could always been improved next time round, as people might not want to hover on the labels

Reply:
Have increased up to 10 pixels and we can see how we get on 👌